### PR TITLE
Fix some errors in the pipeline doc.

### PR DESCRIPTION
**Pull request by abellgithub**

### CODE REVIEW

File: doc/pipeline.rst [hunk 1]
```diff
@@ -354,25 +349,23 @@ then creates the DTM using the :ref:`writers.gdal`.
       "pipeline":[
           "autzen-full.las",
           {
-              "type":"ground",
-              "approximate":true,
+              "type":"filters.pmf",
```
Comment (hobu) [inline doc/pipeline.rst]: we should change this whole block to `filters.smrf`.  We shouldn't encourage `filters.pmf` at all

File: doc/pipeline.rst [hunk 2]
```diff
@@ -161,7 +156,7 @@ reduces memory requirements.
 
 When using :ref:`pdal translate<translate_command>` or
 :ref:`pdal pipeline<pipeline_command>`
-PDAL uses stream mode if possible.  If stream mode can't use used
+PDAL uses stream mode if possible.  If stream mode can't be used
```
Comment (hobu) [inline doc/pipeline.rst]: We should enhance this section to note that one can find out if a stage is streamable or not by looking for the "Streamable Stage" header in any stage document.

**Comment by abellgithub** [comment]
Didn't know we were trying to phase out pmf.  I just picked it because its arguments were the closest to the former ground filter in the example.

Will fix.

**Comment by hobu** [comment]
Not trying to phase it out, but smrf definitely puts the best foot forward for most scenarios

**Pull request merged**